shared_workflow/install_shared.py

```diff
@@ -110,7 +96,7 @@ def install_simulation(
         SimParams.sim_dir.value: sim_dir,
         SimParams.srf_file.value: srf_file,
         SimParams.vm_params.value: vm_params_path,
-        SimParams.sim_duration.value: sim_duration,
+        SimParams.sim_duration.value: vm_params_dict[SimParams.sim_duration.value],
```

Comment:
Is sim duration stored in both vm_params and sim_params?

Reply:
currently yes. We store most of the params from vm_params into sim_params after install, instead of reading it from vm_params when needed. My change was just removing the hard-coded naming/string to use a centralized definition.

Reply:
by tracing the code a bit, it should be removed, as for some reason this was the only parameter that is still stored in sim_params. all other params are stored in vm_params ( and the way we read sim_params is set to read the vm_params by default as well)
At the time of the commit i was just trying to remove the hardcoded naming as i mentioned.